packages/command/src/sfdxFlags.ts

```diff
@@ -150,7 +150,9 @@ function buildStringArray(kind: flags.Kind, options: flags.Array<string>): flags
   const { options: values, validate, ...rest } = options;
   const allowed = new Set(values);
   return option(kind, rest, val => {
-    const vals = val.split(options.delimiter || ',');
+    const regex = /\"(.*?)\"|\'(.*?)\'|,/g;
+    const vals = val.split(regex).filter(i => !!i);
```

Comment:
The regex/split combo will generate some empty strings in the returned array, so the .filter() seems to be the cleanest way to filter them out. Have yet to find a regex pattern that won't generate those empty/undefined entries in the array that .split returns.